<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd"><HTML xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en"><HEAD><META http-equiv="Content-Type" content="text/html; charset=utf8"/><TITLE>MIxS Terms: A quick reference guide</TITLE><LINK rel="schema.DwC" href="http://gensc.org/ns/mixs/terms/"/><META name="DC.title" content="MIxS Terms: A quick reference guide"/><META name="DC.description" content="The quick reference guide to recommended MIxS terms."/><META name="DC.subject" content="biodiversity, standards, genomics"/><META name="DC.creator" content="MIxS Task Group"/><META name="DC.contributor" content="John Wieczorek (MVZ)"/><META name="DC.contributor" content="Pelin Yilmaz (MBI-Bremen)"/><META name="DC.contributor" content="Renzo Kottman (MBI-Bremen)"/><META name="DC.modified" content="2013-03-26"/><META name="DC.dateAccepted" content="2013-03-26"/><META name="DC.format" content="text/html"/><META name="DC.identifier" content="http://gensc.org/ns/mixs/2013-03-26/terms"/><META name="DC.replaces" content="http://gensc.org/ns/mixs/2013-03-26/terms"/><META name="DC.publisher" content="Genomic Standards Consortium"/><META name="DC.rights" content=""/><META name="DC.accessRights" content="public"/><META name="DC.bibliographicCitation" content="MIxS Terms: A quick reference guide. 2013"/><META name="DC.language" content="en"/><LINK rel="meta" href="http://gensc.org/"/><LINK rel="stylesheet" href="../DarwinCore_files/default.css" type="text/css"/><SCRIPT src="../DarwinCore_files/default.js" type="text/javascript"></SCRIPT></HEAD><BODY><TABLE width="100%" cellspacing="0" cellpadding="0" border="0" bgcolor="#617394"><TBODY><TD><A href="http://www.tdwg.org"><IMG src="../DarwinCore_files/GSCLogo.gif"  alt="Genomics Standards Consortium logo"></A></TD><TD width="100%" height="100" align="right" valign="top"></TBODY></TABLE><TABLE width="100%" cellspacing="0" cellpadding="0" border="0">  <TBODY>    <TR>      <TD width="15%" cellspacing="0" cellpadding="0" bgcolor="#617394">        <div id="menu">          <div id="nav" class="topMenu">	    <div class="menuItem level1"><a href="../index.htm" onfocus="blurLink(this);">Introduction</a></div>	    <div class="subMenu"><div class="menuItem"><a href="../index.htm#references" onfocus="blurLink(this);">References</a></div></div>	    <div class="menuItem level1 highlighted"><a href="./index.htm" onfocus="blurLink(this);">Quick Reference Guide</a></div>	    <div class="subMenu"><div class="menuItem"><a href="#theterms" onfocus="blurLink(this);">Term Index</a></div></div>	    <div class="subSubMenu"><div class="menuItem"><a href="#dcindex" onfocus="blurLink(this);">Record-level Terms</a></div></div><div class="separator"><hr class="sub-divider"/></div>	    <div class="subSubMenu"><div class="menuItem"><a href="#occurrenceindex" onfocus="blurLink(this);">Occurrence</a></div></div>	    <div class="subSubMenu"><div class="menuItem"><a href="#eventindex" onfocus="blurLink(this);">Event</a></div></div>	    <div class="subSubMenu"><div class="menuItem"><a href="#locationindex" onfocus="blurLink(this);">Location</a></div></div>	    <div class="subSubMenu"><div class="menuItem"><a href="#investigationindex" onfocus="blurLink(this);">Investigation</a></div></div>	    <div class="subSubMenu"><div class="menuItem"><a href="#environmentindex" onfocus="blurLink(this);">Environment</a></div></div>	    <div class="subSubMenu"><div class="menuItem"><a href="#nucleicacidsequencesourceindex" onfocus="blurLink(this);">Nucleic Acid Sequence Source</a></div></div>	    <div class="subSubMenu"><div class="menuItem"><a href="#sequencingindex" onfocus="blurLink(this);">Sequencing</a></div></div>	    <div class="subMenu"><div class="menuItem"><a href="#termdefs" onfocus="blurLink(this);">Term Definitions</a></div></div>	    <div class="separator"><hr/></div>	    <div class="menuItem level1"><a href="./history/index.htm" onfocus="blurLink(this);">Complete History</a></div>	    <div class="menuItem level1"><a href="./history/decisions/index.htm" onfocus="blurLink(this);">Decision History</a></div>	    <div class="menuItem level1"><a href="./history/versions/index.htm" onfocus="blurLink(this);">Mapping to Old Versions</a></div>          </div>        </div>      </TD>      <TD>        <center><H1>MIxS Terms: A quick reference guide</H1></center>        <TABLE cellspacing="0" class="docinfo">          <TBODY>          <TR>            <TH>Title:</TH>            <TD>MIxS Terms: A quick reference guide</TD>          </TR>          <TR>            <TH>Date Issued:</TH>            <TD>2013-03-26</TD>          </TR>          <TR>            <TH>Date Modified:</TH>            <TD>2013-03-26</TD>          </TR>          <TR>            <TH>Abstract:</TH>            <TD>This document is a quick reference for all recommended MIxS terms. For complete historicalterm information, including version changes and pre-standard terms, see [<A href="./history/index.htm">HISTORY</A>].             </TD>          </TR>          <TR>            <TH>Contributors:</TH>            <TD>John Wieczorek (MVZ)<tuco@berkeley.edu>, Pelin Yilmaz (MPI-Bremen)<pyilmaz@mpi-bremen.de>, Renzo Kottman (MPI-Bremen)<rkottman@mpi-bremen.de></TD>          </TR>          <TR>            <TH>Legal:</TH>            <TD>This document is governed by the standard legal, copyright, licensing provisions and disclaimers issued by the Genomics Stanards Consortium.</TD>          </TR>          <TR>            <TH>Creator:</TH>            <TD>MIxS Task Group</TD>          </TR>          <TR>            <TH>Identifier:</TH>            <TD>http://rs.tdwg.org/dwc/2013-03-26/terms/</TD>          </TR>          <TR>            <TH>Latest Version:</TH>            <TD><A href="http://gensc.org/ns/mixs/terms/index.htm">http://gensc.org/ns/mixs/terms/</A></TD>          </TR>          <TR>            <TH>Replaces:</TH>            <TD><A href="http://gensc.org/ns/mixs/terms/index.htm">http://gensc.org/ns/mixs/terms/</A></TD>          </TR>          <TR>            <TH>Document Status:</TH>            <TD>Draft Standard</TD>          </TR>          </TBODY>        </TABLE>      </TD>    </TR>  </TBODY></TABLE><A name="theterms" id="theterms"></A><H2>1. Term Index</H2><b>Audience</b>: This document is targeted toward those who need a reference (lists and definitions) to the essential information about the current recommended MIxS terms.  <P>This quick guide provides a list of all current terms of the MIxS. The terms areorganized by categories (in bold) in the index. The categories correspond to MIxS terms that are classes (terms that have other terms to describe them). The terms that describe a given class (the class properties) appear in the list immediately below the name of the category in the index. The index provides links to the term descriptions in the table below the index.<P>Names that begin with <i>dcterms:</i> are terms managed by the <i>Dublin Core Metadata Initiative</i> [<a href="http://dublincore.org/">DCMI</a>], which are reproduced here and which may include a description and comments adapted for the biodiversity community. The original definition of such terms can be found by following the link given in the <i>Details</i> section for the term.<P>Names that begin with <i>dwc:</i> are terms managed by the <i>Darwin Core Task Group</i> [<a href="http://rs.tdwg.org/dwc/index.htm/">DWC</a>], which are reproduced here. The original definition of such terms can be found by following the link given in the <i>Details</i> section for the term.<P>Terms under the <i>Record-level Terms</i> section apply to the whole record regardless of the record type.  <P><TABLE class="border" cellspacing="0"><TBODY><!--<A name="dcindex" id="dcindex"></A> <H3>Record-level Terms</H3><P><A name="occurrenceindex" id="occurrenceindex"></A> <H3>Occurrence</H3><P><A name="eventindex" id="eventindex"></A> <H3>Event</H3><P><A name="locationindex" id="locationindex"></A> <H3>dcterms:Location</H3><P><A name="investigationindex" id="investigationindex"></A> <H3>Investigation</H3><P><A name="environmentindex" id="environmentindex"></A> <H3>Environment</H3><P><A name="nucleicacidsequencesourceindex" id="nucleicacidsequencesourceindex"></A> <H3>Nucleic Acid Sequence Source</H3><P><A name="sequencingindex" id="sequencingindex"></A> <H3>Sequencing</H3><P>--><A name="dcindex" id="dcindex"></A> <H3>Record-level Terms</H3><P><a href="#datasetName">datasetName</a>| <a href="#collectionCode">collectionCode</a><A name="occurrenceindex" id="occurrenceindex"></A> <H3>Occurrence</H3><P><a href="#associatedReferences">associatedReferences</a><A name="eventindex" id="eventindex"></A> <H3>Event</H3><P><a href="#eventDate">eventDate</a>| <a href="#samplingProtocol">samplingProtocol</a><A name="locationindex" id="locationindex"></A> <H3>dcterms:Location</H3><P><a href="#verbatimCoordinates">verbatimCoordinates</a>| <a href="#verbatimDepth">verbatimDepth</a>| <a href="#verbatimElevation">verbatimElevation</a>| <a href="#locality">locality</a><A name="investigationindex" id="investigationindex"></A> <H3>Investigation</H3><P><a href="#submittedToINSDC">submittedToINSDC</a>| <a href="#investigationType">investigationType</a>| <a href="#experimentalFactor">experimentalFactor</a><A name="environmentindex" id="environmentindex"></A> <H3>Environment</H3><P><a href="#environmentBiome">environmentBiome</a>| <a href="#environmentFeature">environmentFeature</a>| <a href="#environmentMaterial">environmentMaterial</a>| <a href="#environmentalPackage">environmentalPackage</a><A name="nucleicacidsequencesourceindex" id="nucleicacidsequencesourceindex"></A> <H3>Nucleic Acid Sequence Source</H3><P><a href="#subspecificGeneticLineage">subspecificGeneticLineage</a>| <a href="#ploidy">ploidy</a>| <a href="#numberOfReplicons">numberOfReplicons</a>| <a href="#extrachromosomalElements">extrachromosomalElements</a>| <a href="#estimatedSize">estimatedSize</a>| <a href="#referenceForBiomaterial">referenceForBiomaterial</a>| <a href="#pathogenicity">pathogenicity</a>| <a href="#bioticRelationship">bioticRelationship</a>| <a href="#specificHost">specificHost</a>| <a href="#hostSpecificityOrRange">hostSpecificityOrRange</a>| <a href="#healthDiseaseStatus">healthDiseaseStatus</a>| <a href="#trophicLevel">trophicLevel</a>| <a href="#propagation">propagation</a>| <a href="#encodedTraits">encodedTraits</a>| <a href="#relationshipToOxygen">relationshipToOxygen</a>| <a href="#sampleCollectionDeviceOrMethod">sampleCollectionDeviceOrMethod</a>| <a href="#sampleMaterialProcessing">sampleMaterialProcessing</a>| <a href="#sampleSize">sampleSize</a><A name="sequencingindex" id="sequencingindex"></A> <H3>Sequencing</H3><P><a href="#nucleicAcidExtraction">nucleicAcidExtraction</a>| <a href="#nucleicAcidAmplification">nucleicAcidAmplification</a>| <a href="#librarySize">librarySize</a>| <a href="#libraryReadsSequenced">libraryReadsSequenced</a>| <a href="#libraryConstructionMethod">libraryConstructionMethod</a>| <a href="#libraryVector">libraryVector</a>| <a href="#libraryScreeningStrategy">libraryScreeningStrategy</a>| <a href="#targetGene">targetGene</a>| <a href="#targetSubfragment">targetSubfragment</a>| <a href="#pcrPrimers">pcrPrimers</a>| <a href="#multiplexIdentifiers">multiplexIdentifiers</a>| <a href="#adapters">adapters</a>| <a href="#pcrConditions">pcrConditions</a>| <a href="#sequencingMethod">sequencingMethod</a>| <a href="#sequenceQualityCheck">sequenceQualityCheck</a>| <a href="#chimeraCheck">chimeraCheck</a>| <a href="#assembly">assembly</a>| <a href="#assemblyName">assemblyName</a>| <a href="#finishingStrategy">finishingStrategy</a>| <a href="#annotationSource">annotationSource</a>| <a href="#standardOperatingProcedures">standardOperatingProcedures</a><hr><A name="termdefs" id="termdefs"></A><H2>3. Term Definitions</H2>The table below contains the term information essential for most users. The complete documentation for each termcan be found by following the link in the <i>Details</i> section of the description. The link takes you to the appropriateentry in the <i>Complete History</i> [<A href="./history/index.htm">HISTORY</a>] document. The definitions for the fields describing each term are:<TABLE width="95%" border="0" align="center"><TBODY><TR><TD width="20%"><b>Term Name</b>:</TD><TD width="80%">The name by which the latest version of the term is commonly known.</TD></TR><TR><TD width="20%"><b>Identifier</b>:</TD><TD width="80%">The unique identifier for the latest version of the term.</TD></TR><TR><TD width="20%"><b>Class</b>:</TD><TD width="80%">The primary category of terms (Class) that this term describes.</TD></TR><TR><TD width="20%"><b>Definition</b>:</TD><TD width="80%">A description of the nature of the content of the term.</TD></TR><TR><TD width="20%"><b>Comment</b>:</TD><TD width="80%">Further commentary about the usage of the term, with examples and a reference to further information.</TD></TR><TR><TD width="20%"><b>Details</b>:</TD><TD width="80%">A link to the complete information about the term, including access to its history.</TD></TR></TBODY></TABLE><P><!-- Begin Terms Table --><TABLE cellspacing="0" class="border"><TBODY><TR><TH colspan="2"><A name="Occurrence">Term Name: Occurrence</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://rs.tdwg.org/dwc/terms/Occurrence</TD></TR>	<TR><TD>Class:</TD><TD></TD></TR>	<TR><TD>Definition:</TD><TD>The category of information pertaining to evidence of an occurrence in nature, in a collection, or in a dataset (specimen, observation, etc.).</TD></TR>	<TR><TD>Comment:</TD><TD> For discussion see <a href="http://code.google.com/p/darwincore/wiki/Occurrence">http://code.google.com/p/darwincore/wiki/Occurrence</a></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#Occurrence-2009-04-29">Occurrence</a></TD><TR><TH colspan="2"><A name="Event">Term Name: Event</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://rs.tdwg.org/dwc/terms/Event</TD></TR>	<TR><TD>Class:</TD><TD></TD></TR>	<TR><TD>Definition:</TD><TD>The category of information pertaining to an event (an action that occurs at a place and during a period of time).</TD></TR>	<TR><TD>Comment:</TD><TD> For discussion see <a href="http://code.google.com/p/darwincore/wiki/Event">http://code.google.com/p/darwincore/wiki/Event</a></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#Event-2009-04-29">Event</a></TD><TR><TH colspan="2"><A name="dcterms:Location">Term Name: dcterms:Location</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://purl.org/dc/terms/Location</TD></TR>	<TR><TD>Class:</TD><TD></TD></TR>	<TR><TD>Definition:</TD><TD>A spatial region or named place. For Darwin Core, a set of terms describing a place, whether named or not.</TD></TR>	<TR><TD>Comment:</TD><TD> For discussion see <a href="http://code.google.com/p/darwincore/wiki/Location">http://code.google.com/p/darwincore/wiki/Location</a></TD></TR>	<TR><TD>Details:</TD><TD><a href="http://dublincore.org/documents/dcmi-terms/#classes-Location">http://dublincore.org/documents/dcmi-terms/#classes-Location</a>, <a href="./history/index.htm#dcterms:Location">dcterms:Location</a></TD><TR><TH colspan="2"><A name="Investigation">Term Name: Investigation</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/Investigation</TD></TR>	<TR><TD>Class:</TD><TD></TD></TR>	<TR><TD>Definition:</TD><TD>The category of information pertaining to an investigation.</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#Investigation-2013-03-26">Investigation</a></TD><TR><TH colspan="2"><A name="NucleicAcidSequenceSource">Term Name: NucleicAcidSequenceSource</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Class:</TD><TD></TD></TR>	<TR><TD>Definition:</TD><TD>The category of information pertaining to nucleic acid sequence source.</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#NucleicAcidSequenceSource-2013-03-26">NucleicAcidSequenceSource</a></TD><TR><TH colspan="2"><A name="Enviroment">Term Name: Enviroment</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/Enviroment</TD></TR>	<TR><TD>Class:</TD><TD></TD></TR>	<TR><TD>Definition:</TD><TD>The category of information pertaining to environment.</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#Environment-2013-03-26">Enviroment</a></TD><TR><TH colspan="2"><A name="Sequencing">Term Name: Sequencing</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/Sequencing</TD></TR>	<TR><TD>Class:</TD><TD></TD></TR>	<TR><TD>Definition:</TD><TD>The category of information pertaining to sequencing.</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#Sequencing-2013-03-26">Sequencing</a></TD><TR><TH colspan="2"><A name="datasetName">Term Name: datasetName</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://rs.tdwg.org/dwc/terms/datasetName</TD></TR>	<TR><TD>Class:</TD><TD>all</TD></TR>	<TR><TD>Definition:</TD><TD>The name identifying the data set from which the record was derived.</TD></TR>	<TR><TD>Comment:</TD><TD>Examples: "Grinnell Resurvey Mammals", "Lacey Ctenomys Recaptures". For discussion see <a href="http://code.google.com/p/darwincore/wiki/RecordLevelTerms">http://code.google.com/p/darwincore/wiki/RecordLevelTerms</a></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#datasetName-2009-09-11">datasetName</a></TD><TR><TH colspan="2"><A name="collectionCode">Term Name: collectionCode</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://rs.tdwg.org/dwc/terms/collectionCode</TD></TR>	<TR><TD>Class:</TD><TD>all</TD></TR>	<TR><TD>Definition:</TD><TD>The name, acronym, coden, or initialism identifying the collection or data set from which the record was derived.</TD></TR>	<TR><TD>Comment:</TD><TD>Examples: "Mammals", "Hildebrandt", "eBird". For discussion see <a href="http://code.google.com/p/darwincore/wiki/RecordLevelTerms">http://code.google.com/p/darwincore/wiki/RecordLevelTerms</a></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#collectionCode-2009-09-11">collectionCode</a></TD><TR><TH colspan="2"><A name="associatedReferences">Term Name: associatedReferences</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://rs.tdwg.org/dwc/terms/associatedReferences</TD></TR>	<TR><TD>Class:</TD><TD>http://rs.tdwg.org/dwc/terms/Occurrence</TD></TR>	<TR><TD>Definition:</TD><TD>A list (concatenated and separated) of identifiers (publication, bibliographic reference, global unique identifier, URI) of literature associated with the Occurrence.</TD></TR>	<TR><TD>Comment:</TD><TD>Examples: "http://www.sciencemag.org/cgi/content/abstract/322/5899/261", "Christopher J. Conroy, Jennifer L. Neuwald. 2008. Phylogeographic study of the California vole, Microtus californicus Journal of Mammalogy, 89(3):755-767.". For discussion see <a href="http://code.google.com/p/darwincore/wiki/Occurrence">http://code.google.com/p/darwincore/wiki/Occurrence</a></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#associatedReferences-2009-04-24">associatedReferences</a></TD><TR><TH colspan="2"><A name="eventDate">Term Name: eventDate</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://rs.tdwg.org/dwc/terms/eventDate</TD></TR>	<TR><TD>Class:</TD><TD>http://rs.tdwg.org/dwc/terms/Event</TD></TR>	<TR><TD>Definition:</TD><TD>The date-time or interval during which an Event occurred. For occurrences, this is the date-time when the event was recorded. Not suitable for a time in a geological context. Recommended best practice is to use an encoding scheme, such as ISO 8601:2004(E).</TD></TR>	<TR><TD>Comment:</TD><TD>Examples: "1963-03-08T14:07-0600" is 8 Mar 1963 2:07pm in the time zone six hours earlier than UTC, "2009-02-20T08:40Z" is 20 Feb 2009 8:40am UTC, "1809-02-12" is 12 Feb 1809, "1906-06" is Jun 1906, "1971" is just that year, "2007-03-01T13:00:00Z/2008-05-11T15:30:00Z" is the interval between 1 Mar 2007 1pm UTC and 11 May 2008 3:30pm UTC, "2007-11-13/15" is the interval between 13 Nov 2007 and 15 Nov 2007. For discussion see <a href="http://code.google.com/p/darwincore/wiki/Event">http://code.google.com/p/darwincore/wiki/Event</a></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#eventDate-2009-04-24">eventDate</a></TD><TR><TH colspan="2"><A name="samplingProtocol">Term Name: samplingProtocol</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://rs.tdwg.org/dwc/terms/samplingProtocol</TD></TR>	<TR><TD>Class:</TD><TD>http://rs.tdwg.org/dwc/terms/Event</TD></TR>	<TR><TD>Definition:</TD><TD>The name of, reference to, or description of the method or protocol used during an Event.</TD></TR>	<TR><TD>Comment:</TD><TD>Examples: "UV light trap", "mist net", "bottom trawl", "ad hoc observation", "point count", "Penguins from space: faecal stains reveal the location of emperor penguin colonies, http://dx.doi.org/10.1111/j.1466-8238.2009.00467.x", "Takats et al. 2001. Guidelines for Nocturnal Owl Monitoring in North America. Beaverhill Bird Observatory and Bird Studies Canada, Edmonton, Alberta. 32 pp.", "http://www.bsc-eoc.org/download/Owl.pdf". For discussion see <a href="http://code.google.com/p/darwincore/wiki/Event">http://code.google.com/p/darwincore/wiki/Event</a></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#samplingProtocol-2009-04-24">samplingProtocol</a></TD><TR><TH colspan="2"><A name="verbatimCoordinates">Term Name: verbatimCoordinates</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://rs.tdwg.org/dwc/terms/verbatimCoordinates</TD></TR>	<TR><TD>Class:</TD><TD>http://purl.org/dc/terms/Location</TD></TR>	<TR><TD>Definition:</TD><TD>The verbatim original spatial coordinates of the Location. The coordinate ellipsoid, geodeticDatum, or full Spatial Reference System (SRS) for these coordinates should be stored in verbatimSRS and the coordinate system should be stored in verbatimCoordinateSystem.</TD></TR>	<TR><TD>Comment:</TD><TD>Examples: "41 05 54S 121 05 34W", "17T 630000 4833400". For discussion see <a href="http://code.google.com/p/darwincore/wiki/Location">http://code.google.com/p/darwincore/wiki/Location</a></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#verbatimCoordinates-2009-04-24">verbatimCoordinates</a></TD><TR><TH colspan="2"><A name="verbatimDepth">Term Name: verbatimDepth</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://rs.tdwg.org/dwc/terms/verbatimDepth</TD></TR>	<TR><TD>Class:</TD><TD>http://purl.org/dc/terms/Location</TD></TR>	<TR><TD>Definition:</TD><TD>The original description of the depth below the local surface.</TD></TR>	<TR><TD>Comment:</TD><TD>Example: "100-200 m". For discussion see <a href="http://code.google.com/p/darwincore/wiki/Location">http://code.google.com/p/darwincore/wiki/Location</a></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#verbatimDepth-2009-04-24">verbatimDepth</a></TD><TR><TH colspan="2"><A name="verbatimElevation">Term Name: verbatimElevation</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://rs.tdwg.org/dwc/terms/verbatimElevation</TD></TR>	<TR><TD>Class:</TD><TD>http://purl.org/dc/terms/Location</TD></TR>	<TR><TD>Definition:</TD><TD>The original description of the elevation (altitude, usually above sea level) of the Location.</TD></TR>	<TR><TD>Comment:</TD><TD>Example: "100-200 m". For discussion see <a href="http://code.google.com/p/darwincore/wiki/Location">http://code.google.com/p/darwincore/wiki/Location</a></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#verbatimElevation-2009-04-24">verbatimElevation</a></TD><TR><TH colspan="2"><A name="locality">Term Name: locality</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://rs.tdwg.org/dwc/terms/locality</TD></TR>	<TR><TD>Class:</TD><TD>http://purl.org/dc/terms/Location</TD></TR>	<TR><TD>Definition:</TD><TD>The specific description of the place. Less specific geographic information can be provided in other geographic terms (higherGeography, continent, country, stateProvince, county, municipality, waterBody, island, islandGroup). This term may contain information modified from the original to correct perceived errors or standardize the description.</TD></TR>	<TR><TD>Comment:</TD><TD>Example: "Bariloche, 25 km NNE via Ruta Nacional 40 (=Ruta 237)". For discussion see <a href="http://code.google.com/p/darwincore/wiki/Location">http://code.google.com/p/darwincore/wiki/Location</a></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#locality-2009-04-24">locality</a></TD><TR><TH colspan="2"><A name="submittedToINSDC">Term Name: submittedToINSDC</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/submittedToINSDC</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Investigation</TD></TR>	<TR><TD>Definition:</TD><TD>Depending on the study (large-scale e.g. done with next generation sequencing technology, or small-scale) sequences have to be submitted to SRA (Sequence Read Archive), DRA (DDBJ Read Archive) or via the classical Webin/Sequin systems to Genbank, ENA and DDBJ.Although this field is mandatory, it is meant as a "self-test" field, therefore it is not necessary to include this field in contextual data submitted to databases</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#submitted_to_insdc-2011-01-26">submittedToINSDC</a></TD><TR><TH colspan="2"><A name="investigationType">Term Name: investigationType</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/investigationType</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Investigation</TD></TR>	<TR><TD>Definition:</TD><TD>Nucleic Acid Sequence Report is the root element of all MIGS/MIMS compliant reports as standardized by Genomic Standards Consortium. This field is either eukaryote,bacteria,virus,plasmid,organelle, metagenome, miens-survey or miens-specimen</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#investigation_type-2011-01-26">investigationType</a></TD><TR><TH colspan="2"><A name="experimentalFactor">Term Name: experimentalFactor</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/experimentalFactor</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Investigation</TD></TR>	<TR><TD>Definition:</TD><TD>Experimental factors are essentially the variable aspects of an experiment design which can be used to describe an experiment, or set of experiments, in an increasingly detailed manner. This field accepts ontology terms from Experimental Factor Ontology (EFO) and/or Ontology for Biomedical Investigations (OBI). For a browser of EFO (v132) terms, please see http://bioportal.bioontology.org/visualize/44641; for a browser of OBI (v1.0) terms please see http://bioportal.bioontology.org/visualize/40832</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#experimental_factor-2011-01-26">experimentalFactor</a></TD><TR><TH colspan="2"><A name="environmentBiome">Term Name: environmentBiome</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/environmentBiome</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Environment</TD></TR>	<TR><TD>Definition:</TD><TD>In environmental biome level are the major classes of ecologically similar communities of plants, animals, and other organisms. Biomes are defined based on factors such as plant structures, leaf types, plant spacing, and other factors like climate. Examples include: desert, taiga, deciduous woodland, or coral reef. EnvO (v1.53) terms listed under environmental biome can be found from the link: http://bioportal.bioontology.org/visualize/44405/?conceptid=ENVO%3A00000428</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#env_biome-2011-01-26">environmentBiome</a></TD><TR><TH colspan="2"><A name="environmentFeature">Term Name: environmentFeature</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/environmentFeature</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Environment</TD></TR>	<TR><TD>Definition:</TD><TD>Environmental feature level includes geographic environmental features. Examples include: harbor, cliff, or lake. EnvO (v1.53) terms listed under environmental feature can be found from the link: http://bioportal.bioontology.org/visualize/44405/?conceptid=ENVO%3A00002297</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#env_feature-2011-01-26">environmentFeature</a></TD><TR><TH colspan="2"><A name="environmentMaterial">Term Name: environmentMaterial</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/environmentMaterial</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Environment</TD></TR>	<TR><TD>Definition:</TD><TD>The environmental material level refers to the matter that was displaced by the sample, prior to the sampling event. Environmental matter terms are generally mass nouns. Examples include: air, soil, or water. EnvO (v1.53) terms listed under environmental matter can be found from the link: http://bioportal.bioontology.org/visualize/44405/?conceptid=ENVO%3A00010483</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#env_material-2011-01-26">environmentMaterial</a></TD><TR><TH colspan="2"><A name="environmentalPackage">Term Name: environmentalPackage</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/environmentalPackage</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Environment</TD></TR>	<TR><TD>Definition:</TD><TD>MIGS/MIMS/MIENS extension for reporting of measurements and observations obtained from one or more of the environments where the sample was obtained. All environmental packages listed here are further defined in separate subtables. By giving the name of the environmental package, a selection of fields can be made from the subtables and can be reported</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#env_package-2011-01-26">environmentalPackage</a></TD><TR><TH colspan="2"><A name="subspecificGeneticLineage">Term Name: subspecificGeneticLineage</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/subspecificGeneticLineage</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>This should provide further information about the genetic distinctness of this lineage by recording additional information i.e biovar, serovar, serotype, biovar, or any relevant genetic typing schemes like Group I plasmid. It can also contain alternative taxonomic information</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#subspecf_gen_lin-2011-01-26">subspecificGeneticLineage</a></TD><TR><TH colspan="2"><A name="ploidy">Term Name: ploidy</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/ploidy</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>The ploidy level of the genome (e.g. allopolyploid, haploid, diploid, triploid, tetraploid). It has implications for the downstream study of duplicated gene and regions of the genomes (and perhaps for difficulties in assembly). For terms, please select terms listed under class ploidy (PATO:001374) of Phenotypic Quality Ontology (PATO), and for a browser of PATO (v1.269) please refer to http://bioportal.bioontology.org/visualize/44601/?conceptid=PATO%3A0001374</TD></TR>	<TR><TD>Comment:</TD><TD>allopolyploid, polyploid</TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#ploidy-2011-01-26">ploidy</a></TD><TR><TH colspan="2"><A name="numberOfReplicons">Term Name: numberOfReplicons</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/numberOfReplicons</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>Reports the number of replicons in a nuclear genome of eukaryotes, in the genome of a bacterium or archaea or the number of segments in a segmented virus. Always applied to the haploid chromosome count of a eukaryote</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#num_replicons-2011-01-26">numberOfReplicons</a></TD><TR><TH colspan="2"><A name="extrachromosomalElements">Term Name: extrachromosomalElements</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/extrachromosomalElements</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>Do plasmids exist of significant phenotypic consequence (e.g. ones that determine virulence or antibiotic resistance). Megaplasmids? Other plasmids (borrelia has 15+ plasmids)</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#extrachrom_elements-2011-01-26">extrachromosomalElements</a></TD><TR><TH colspan="2"><A name="estimatedSize">Term Name: estimatedSize</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/estimatedSize</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>The estimated size of the genome prior to sequencing. Of particular importance in the sequencing of (eukaryotic) genome which could remain in draft form for a long or unspecified period.</TD></TR>	<TR><TD>Comment:</TD><TD>300000 bp</TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#estimated_size-2011-01-26">estimatedSize</a></TD><TR><TH colspan="2"><A name="referenceForBiomaterial">Term Name: referenceForBiomaterial</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/referenceForBiomaterial</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>primary publication if isolated before genome publication; otherwise, primary genome report</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#ref_biomaterial-2011-01-26">referenceForBiomaterial</a></TD><TR><TH colspan="2"><A name="pathogenicity">Term Name: pathogenicity</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/pathogenicity</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>To what is the entity pathogenic.</TD></TR>	<TR><TD>Comment:</TD><TD>human, animal, plant, fungi, bacteria</TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#pathogenicity-2011-01-26">pathogenicity</a></TD><TR><TH colspan="2"><A name="bioticRelationship">Term Name: bioticRelationship</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/bioticRelationship</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>Is it free-living or in a host and if the latter what type of relationship is observed</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#bioticRelationship-2011-01-26">bioticRelationship</a></TD><TR><TH colspan="2"><A name="specificHost">Term Name: specificHost</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/specificHost</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>If there is a host involved, please provide its taxid (or environmental if not actually isolated from the dead or alive host - i.e. pathogen could be isolated from a swipe of a bench etc) and report whether it is a laboratory or natural host). From this we can calculate any number of groupings of hosts (e.g. animal vs plant, all fish hosts, etc)</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#specific_host-2011-01-26">specificHost</a></TD><TR><TH colspan="2"><A name="hostSpecificityOrRange">Term Name: hostSpecificityOrRange</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/hostSpecificityOrRange</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>The NCBI taxonomy identifier of the specific host if it is known</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#host_spec_range-2011-01-26">hostSpecificityOrRange</a></TD><TR><TH colspan="2"><A name="healthDiseaseStatus">Term Name: healthDiseaseStatus</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/healthDiseaseStatus</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>Health or disease status of specific host at time of collection. This field accepts PATO (v1.269) terms, for a browser please see http://bioportal.bioontology.org/visualize/44601</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#health_disease_stat-2011-01-26">healthDiseaseStatus</a></TD><TR><TH colspan="2"><A name="trophicLevel">Term Name: trophicLevel</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/trophicLevel</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>Trophic levels are the feeding position in a food chain. Microbes can be a range of producers (e.g. chemolithotroph)</TD></TR>	<TR><TD>Comment:</TD><TD>"</TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#trophic_level-2011-01-26">trophicLevel</a></TD><TR><TH colspan="2"><A name="propagation">Term Name: propagation</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/propagation</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>This field is specific to different taxa. For phages: lytic/lysogenic, for plasmids: incompatibility group (Note: there is the strong opinion to name phage propagation obligately lytic or temperate, therefore we also give this choice</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#propagation-2011-01-26">propagation</a></TD><TR><TH colspan="2"><A name="encodedTraits">Term Name: encodedTraits</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/encodedTraits</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>Should include key traits like antibiotic resistance or xenobiotic degradation phenotypes for plasmids, converting genes for phage</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#encoded_traits-2011-01-26">encodedTraits</a></TD><TR><TH colspan="2"><A name="relationshipToOxygen">Term Name: relationshipToOxygen</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/relationshipToOxygen</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>Is this organism an aerobe, anaerobe? Please note that aerobic and anaerobic are valid descriptors for microbial environments</TD></TR>	<TR><TD>Comment:</TD><TD>'</TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#rel_to_oxygen-2011-01-26">relationshipToOxygen</a></TD><TR><TH colspan="2"><A name="sampleCollectionDeviceOrMethod">Term Name: sampleCollectionDeviceOrMethod</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/sampleCollectionDeviceOrMethod</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>The method or device employed for collecting the sample</TD></TR>	<TR><TD>Comment:</TD><TD>biopsy, niskin bottle, push core</TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#samp_collect_device-2011-01-26">sampleCollectionDeviceOrMethod</a></TD><TR><TH colspan="2"><A name="sampleMaterialProcessing">Term Name: sampleMaterialProcessing</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/sampleMaterialProcessing</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>Any processing applied to the sample during or after retrieving the sample from environment. This field accepts OBI, for a browser of OBI (v1.0) terms please see http://bioportal.bioontology.org/visualize/40832</TD></TR>	<TR><TD>Comment:</TD><TD>filtering of seawater, storing samples in ethanol</TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#samp_mat_process-2011-01-26">sampleMaterialProcessing</a></TD><TR><TH colspan="2"><A name="sampleSize">Term Name: sampleSize</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/sampleSize</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/NucleicAcidSequenceSource</TD></TR>	<TR><TD>Definition:</TD><TD>Amount or size of sample (volume, mass or area) that was collected</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#samp_size-2011-01-26">sampleSize</a></TD><TR><TH colspan="2"><A name="nucleicAcidExtraction">Term Name: nucleicAcidExtraction</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/nucleicAcidExtraction</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>Link to a literature reference, electronic resource or a standard operating procedure (SOP)</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#nucl_acid_ext-2011-01-26">nucleicAcidExtraction</a></TD><TR><TH colspan="2"><A name="nucleicAcidAmplification">Term Name: nucleicAcidAmplification</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/nucleicAcidAmplification</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>Link to a literature reference, electronic resource or a standard operating procedure (SOP)</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#nucl_acid_amp-2011-01-26">nucleicAcidAmplification</a></TD><TR><TH colspan="2"><A name="librarySize">Term Name: librarySize</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/librarySize</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>Total number of clones in the library prepared for the project</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#lib_size-2011-01-26">librarySize</a></TD><TR><TH colspan="2"><A name="libraryReadsSequenced">Term Name: libraryReadsSequenced</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/libraryReadsSequenced</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>Total number of clones sequenced from the library</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#lib_reads_seqd-2011-01-26">libraryReadsSequenced</a></TD><TR><TH colspan="2"><A name="libraryConstructionMethod">Term Name: libraryConstructionMethod</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/libraryConstructionMethod</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>Library construction method used for clone libraries</TD></TR>	<TR><TD>Comment:</TD><TD>paired-end,single,vector</TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#lib_const_meth-2011-01-26">libraryConstructionMethod</a></TD><TR><TH colspan="2"><A name="libraryVector">Term Name: libraryVector</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/libraryVector</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>Cloning vector type(s) used in construction of libraries</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#lib_vector-2011-01-26">libraryVector</a></TD><TR><TH colspan="2"><A name="libraryScreeningStrategy">Term Name: libraryScreeningStrategy</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/libraryScreeningStrategy</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>Specific enrichment or screening methods applied before and/or after creating clone libraries</TD></TR>	<TR><TD>Comment:</TD><TD>enriched, screened, normalized</TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#lib_screen-2011-01-26">libraryScreeningStrategy</a></TD><TR><TH colspan="2"><A name="targetGene">Term Name: targetGene</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/targetGene</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>Targeted gene or locus name for marker gene studies</TD></TR>	<TR><TD>Comment:</TD><TD>16S rRNA, 18S rRNA, nif, amoA, rpo</TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#target_gene-2011-01-26">targetGene</a></TD><TR><TH colspan="2"><A name="targetSubfragment">Term Name: targetSubfragment</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/targetSubfragment</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>Name of subfragment of a gene or locus. Important to e.g. identify special regions on marker genes like V6 on 16S rRNA</TD></TR>	<TR><TD>Comment:</TD><TD>V6, V9, ITS</TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#target_subfragment-2011-01-26">targetSubfragment</a></TD><TR><TH colspan="2"><A name="pcrPrimers">Term Name: pcrPrimers</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/pcrPrimers</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>PCR primers that were used to amplify the sequence of the targeted gene, locus or subfragment. This field should contain all the primers used for a single PCR reaction if multiple forward or reverse primers are present in a single PCR reaction. The primer sequence should be reported in uppercase letters</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#pcr_primers-2011-01-26">pcrPrimers</a></TD><TR><TH colspan="2"><A name="multiplexIdentifiers">Term Name: multiplexIdentifiers</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/multiplexIdentifiers</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>Molecular barcodes, called Multiplex Identifiers (MIDs), that are used to specifically tag unique samples in a sequencing run. Sequence should be reported in uppercase letters</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#mid-2011-01-26">multiplexIdentifiers</a></TD><TR><TH colspan="2"><A name="adapters">Term Name: adapters</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/adapters</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>Adapters provide priming sequences for both amplification and sequencing of the sample-library fragments. Both adapters should be reported; in uppercase letters</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#adapters-2011-01-26">adapters</a></TD><TR><TH colspan="2"><A name="pcrConditions">Term Name: pcrConditions</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/pcrConditions</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>Description of reaction conditions and components for PCR in the form of  'initial denaturation:94degC_1.5min; annealing=...'</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#pcr_cond-2011-01-26">pcrConditions</a></TD><TR><TH colspan="2"><A name="sequencingMethod">Term Name: sequencingMethod</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/sequencingMethod</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>Sequencing method used; e.g. Sanger, pyrosequencing, ABI-solid</TD></TR>	<TR><TD>Comment:</TD><TD>Sanger dideoxysequencing, pyrosequencing, polony</TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#seq_meth-2011-01-26">sequencingMethod</a></TD><TR><TH colspan="2"><A name="sequenceQualityCheck">Term Name: sequenceQualityCheck</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/sequenceQualityCheck</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>Indicate if the sequence has been called by automatic systems (none) or undergone a manual editing procedure (e.g. by inspecting the raw data or chromatograms). Applied only for sequences that are not submitted to SRA,ENA or DRA</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#seq_quality_check-2011-01-26">sequenceQualityCheck</a></TD><TR><TH colspan="2"><A name="chimeraCheck">Term Name: chimeraCheck</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/chimeraCheck</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>A chimeric sequence, or chimera for short, is a sequence comprised of two or more phylogenetically distinct parent sequences. Chimeras are usually PCR artifacts thought to occur when a prematurely terminated amplicon reanneals to a foreign DNA strand and is copied to completion in the following PCR cycles. The point at which the chimeric sequence changes from one parent to the next is called the breakpoint or conversion point</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#chimera_check-2011-01-26">chimeraCheck</a></TD><TR><TH colspan="2"><A name="assembly">Term Name: assembly</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/assembly</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>How was the assembly done (e.g. with a text based assembler like phrap or a flowgram assembler); estimated error rate associated with the finished sequences (e.g. error rate of 1 in 1000 bp); and the method of calculation</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#assembly-2011-01-26">assembly</a></TD><TR><TH colspan="2"><A name="assemblyName">Term Name: assemblyName</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/assemblyName</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>Name/version of the assembly provided by the submitter that is used in the genome browsers and in the community</TD></TR>	<TR><TD>Comment:</TD><TD>HuRef, JCVI_ISG_i3_1.0</TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#assembly_name-2011-01-26">assemblyName</a></TD><TR><TH colspan="2"><A name="finishingStrategy">Term Name: finishingStrategy</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/finishingStrategy</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>Was the genome project intended to produce a complete or draft genome, Coverage, the fold coverage of the sequencing expressed as 2x, 3x, 18x  etc, and how many contigs were produced for the genome</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#finishing_strategy-2011-01-26">finishingStrategy</a></TD><TR><TH colspan="2"><A name="annotationSource">Term Name: annotationSource</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/annotationSource</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>For cases where annotation was provided by a community jamboree or model organism database rather than by a specific submitter</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#annot_source-2011-01-26">annotationSource</a></TD><TR><TH colspan="2"><A name="standardOperatingProcedures">Term Name: standardOperatingProcedures</A></TH></TR>	<TR><TD>Identifier:</TD><TD>http://gensc.org/ns/mixs/standardOperatingProcedures</TD></TR>	<TR><TD>Class:</TD><TD>http://gensc.org/ns/mixs/terms/Sequencing</TD></TR>	<TR><TD>Definition:</TD><TD>Standard operating procedures used in assembly and/or annotation of genomes, metagenomes or environmental sequences</TD></TR>	<TR><TD>Comment:</TD><TD></TD></TR>	<TR><TD>Details:</TD><TD><a href="./history/index.htm#sop-2011-01-26">standardOperatingProcedures</a></TD><!-- End Terms Table --></TBODY></TABLE><!-- Footer --><hr><p><a rel="license" href="http://creativecommons.org/licenses/by/3.0/us/">    <img alt="Creative Commons License" id="creative_commons_icon" src="http://i.creativecommons.org/l/by/3.0/88x31.png" /></a>    Copyright 2009-2011 - Biodiversity Information Standards - TDWG - <a href="http://www.tdwg.org/about-tdwg/contact-us/">Contact Us</a><br/> <p>Except where otherwise noted, content on this site is licensed under a    <a rel="license" href="http://creativecommons.org/licenses/by/3.0/us/"> Creative Commons    Attribution 3.0 United States License</a>.</p> <!-- The following is used to register Google Analytics for Darwin Core --> <script src="http://www.google-analytics.com/urchin.js" type="text/javascript"> </script> <script type="text/javascript"> _uacct = "UA-386033-4";urchinTracker();</script> </BODY></HTML>	